pkgs/test_core/lib/src/runner/reporter/expanded.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
@jakemac53 - this import is a problem. I think we use the expanded reporter by default on the web when you invoke a `_test.dart` file directly rather than go through the runner.

I think I may need to go back to `Sink<List<int>>`. Any concerns?

Reply:
On second though, that will cause a problem here:

https://github.com/dart-lang/test/blob/5a3d81a1235a8f515064e3514f98f5aa7c3e5721/pkgs/test_core/lib/test_core.dart#L62-L63

I think it might not work to add an import to `dart:io` there?